### DISCUSSION TITLE
Why has the editing box for content keywords or non-included keywords disappeared in the SkyWalking log module?

### DISCUSSION THREAD

**Discussion by poisonduyao** [apache/skywalking#11451] [Q&A]
Currently, I am using Elasticsearch as storage, and this edit box occasionally appears. Could this be a bug or a configuration issue?
![image](https://github.com/apache/skywalking/assets/26665639/4b4ad7b8-6352-4959-b8bd-e217e027c610)

**Comment by wu-sheng** [comment]
Is your page wide too small?

**Reply by poisonduyao** [reply]
![image](https://github.com/apache/skywalking/assets/26665639/b497f0b4-71f0-42cf-8017-8f28ae61adea)  
It just feels like a normal page

**Reply by wu-sheng** [reply]
It never happened at my side. @poisonduyao Are you using the latest version?

@Fine0830 Any clue?

**Reply by poisonduyao** [reply]
The current version I am using is 9.3.0-7ad0406.

**Comment by Fine0830** [comment]
I think your `supportQueryLogsByKeywords` is false.

**Reply by poisonduyao** [reply]
I couldn't find this configuration item.

**Reply by wu-sheng** [reply]
This is not a config, it is determined by storage option.
Only elasticsearch storage supports this.

**Reply by poisonduyao** [reply]
This option doesn't seem to exist in the default configuration file. Even after I added it, there was no effect. However, another configuration with the same settings allows the edit box to appear. What could be the reason for this

**Reply by wu-sheng** [reply]
I think you don't follow the context. We mean, if you choose Elasticsearch as storage, this will show up.
